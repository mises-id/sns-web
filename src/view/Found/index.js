import React, { useReducer, useEffect, useRef, useState } from 'react';
import { Cell ,Radio} from 'zarm';
const Found = (props)=>{
  const [multiple, setmultiple] = useState(true)
  return <div>
    <Cell title="是否多选">
        <Radio.Group
          compact
          type="button"
          value={multiple}
          onChange={(value) => {
            setmultiple(value);
          }}
        >
          <Radio value={false}>单选</Radio>
          <Radio value={true}>双选</Radio>
        </Radio.Group>
      </Cell>
  </div>
}
export default Found