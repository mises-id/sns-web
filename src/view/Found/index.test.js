import React from 'react';
import renderer from 'react-test-renderer';
import Found from './';
jest.mock('react-i18next', () => ({
  useTranslation: () => ({t: key => key})
}));
describe('Found', () => {
  it('should render correctly', () => {
    const wrapper = renderer.create(<Found></Found>).toJSON()
    expect(wrapper).toMatchSnapshot();
  });
});