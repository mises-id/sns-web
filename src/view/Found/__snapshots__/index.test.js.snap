// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Found should render correctly 1`] = `
<div>
  <Cell
    disabled={false}
    hasArrow={false}
    prefixCls="za-cell"
    title="是否多选"
  >
    <RadioGroup
      block={false}
      compact={true}
      disabled={false}
      ghost={false}
      onChange={[Function]}
      prefixCls="za-radio-group"
      shape="radius"
      size="xs"
      type="button"
      value={true}
    >
      <Radio
        disabled={false}
        prefixCls="za-radio"
        shape="radius"
        value={false}
      >
        单选
      </Radio>
      <Radio
        disabled={false}
        prefixCls="za-radio"
        shape="radius"
        value={true}
      >
        双选
      </Radio>
    </RadioGroup>
  </Cell>
</div>
`;
